### PR TITLE
DetailsRow: pass aria-label to row's checkbox

### PR DESCRIPTION
**Pull request by Vitalius1**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes a Teams reported bug
- [X] Include a change request file using `$ npm run change`

#### Description of changes

Checkboxes on each row were passed the `checkButtonAriaLabel` prop but it was assigned only to the `title` attribute. This PR ensures that the same prop is assigned to `aria-label` too.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9751)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Basic.Example.tsx [hunk 1]
```diff
@@ -68,6 +68,7 @@ export class DetailsListBasicExample extends React.Component<{}, IDetailsListBas
         />
         <MarqueeSelection selection={this._selection}>
           <DetailsList
+            checkButtonAriaLabel="Row checkbox"
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Basic.Example.tsx]: Is there a reason you only added a `checkButtonAriaLabel` value to the basic example? :)

File: packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.base.tsx [hunk 2]
```diff
@@ -336,7 +336,7 @@ export class DetailsRowBase extends BaseComponent<IDetailsRowBaseProps, IDetails
   }
 
   protected _onRenderCheck(props: IDetailsRowCheckProps) {
-    return <DetailsRowCheck {...props} />;
+    return <DetailsRowCheck {...props} aria-label={props.title} />;
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.base.tsx]: Instead, you could probably add

```
"aria-label": checkButtonAriaLabel,
```

to the `onRenderCheck` method call on line 271

Comment (Vitalius1) [reply packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.base.tsx]: Updated.

File: packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Basic.Example.tsx (hunk 1 above)
Comment (Vitalius1) [reply packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Basic.Example.tsx]: Added on every example.

**Comment by Vitalius1** [comment]
@KevinTCoughlin I’ve updated the PR with new commits. Can you take another look please? Thanks!

**Comment by natalieethell** [comment]
When we remove 'title', I believe we remove the native tooltip that it provides. Do we think that this change in behavior will break any partners expecting the native tooltip?

**Comment by Vitalius1** [comment]
@natalieethell I asked Ben about it and he agrees that for visual users that tooltip is not providing any valuable information as it’s very obvious what that element represents. Narrator announces only once even if both are present but to be on the safe side for other accessibility tools we are removing the title to avoid a double reading of the same thing.

**Comment by Vitalius1** [comment]
@KevinTCoughlin, @natalieethell actually I looked at what we have today on OneDrive and it looks like they are setting that prop and the tooltip is popping up. I will follow up with Ben and see if that was intended or just a byproduct of setting that prop.


@betrue-final-final need your input. Thanks!

**Comment by betrue-final-final** [comment]
The tooltip visual is unnecessary.

**Pull request merged**